app/assets/javascripts/tour.js

```diff
@@ -0,0 +1,53 @@
+function tour_firstAnswer() {
```

Comment:
Don't mix snake_case and camelCase - use camelCase for JS, i.e. tourFirstAnswer. If you want to namespace them, wrap everything in a `tour` object, i.e.

```js
const tour = {
  firstAnswer: function () { ... }
};
```